### DISCUSSION TITLE
How to change button state by code?

### DISCUSSION THREAD

**Discussion by RobinsonLaw** [alejandroautalan/pygubu-designer#171] [Q&A] [answered]
for example not success
self.button_start.config(state="DISABLE")

**Comment by alejandroautalan** [comment]
Hello @RobinsonLaw thanks for trying pygubu.

You can change button state with:

```python
self.button_start.configure(state="disabled")
```

Buttons from ttk can be in "normal" or "disabled" state.

Regards
Alejandro A.

**Comment by RobinsonLaw** [comment, marked as answer]
thank you I got it

Alejandro Autalán ***@***.***> 於 2022年12月4日週日 上午7:13寫道：
